<launch>
    <!-- 启动ugv_control_node -->
    <node pkg="sunray_swarm" type="ugv_control_node" name="ugv_control_node_1" output="screen">
        <param name="agent_id" value="1" />
        <rosparam command="load" file="$(find sunray_swarm)/launch_sim/ugv_params.yaml" />
    </node>

    <!-- 启动ugv_sim_node -->
    <node pkg="sunray_swarm" type="agent_sim_node" name="agent_sim_node_1" output="screen">
        <param name="agent_id" value="1" />
        <param name="init_pos_x" value="-2.0" />
        <param name="init_pos_y" value="4.0" />
        <rosparam command="load" file="$(find sunray_swarm)/launch_sim/ugv_params.yaml" />
    </node>

    <!-- Rviz config -->
    <arg name="rviz_enable" default="true"/>
    <arg name="rivz_config" default="$(find sunray_swarm)/launch_sim/single_agent.rviz"/>
	<!-- 启动Rviz-->
	<group if="$(arg rviz_enable)">
        <node type="rviz" name="rviz" pkg="rviz" args="-d $(arg rivz_config)"/>
    </group>

</launch>

